### PR TITLE
Notify user while adding an existing group again

### PR DESCRIPTION
**Pull request by sharidas**
When user tries to add group from UI which already
exists, then a notification in the UI will help user
understand the cause for not adding.

Signed-off-by: Sujith H <sharidasan@owncloud.com>

## Description

Provide error notification if the group already exist when user tries to add group in the UI.

## Related Issue

https://github.com/owncloud/core/issues/18972

## Motivation and Context

Provide error notification when user tries to add group already available

## Screenshots (if appropriate):
errormessage

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by sharidas** [comment]
I need to see if this JS is called only for group addition multiselect or any other place in the UI.

File: core/js/multiselect.js [hunk 1]
```diff
@@ -231,6 +231,7 @@
 							$.each(options,function(index, item) {
 								if ($(item).val() == value || $(item).text() == value) {
 									exists = true;
+									OC.Notification.showTemporary(t('settings', "Error adding group: Group already exist"));
```
Comment (phil-davis) [inline core/js/multiselect.js]: s/already exist/already exists/

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by phil-davis** [comment]
Backport stable10 #30194

**Pull request merged**